_includes/_references.md

```diff
@@ -14,6 +17,7 @@
 [Warren Togami]: https://github.com/wtogami
 [Wladimir van der Laan]: https://github.com/laanwj
 [Jorge Timon]: https://github.com/jtimon
```

Comment:
It's technically correct although a misspelling of his name. If this is removed all references to this would need to be changed at the same time.